packages/klick/PKGBUILD

```diff
@@ -0,0 +1,30 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Daniel Appelt <daniel.appelt@gmail.com>
+# Contributor: Christoph Zeiler <rabyte*gmail>
+# Contributor: Philipp Überbacher <murks at lavabit dot com>
+# Contributor: Christopher Arndt <aur -at- chrisarndt -dot- de>
+
+pkgname=klick
+pkgver=0.14.2
+pkgrel=1
+pkgdesc='An advanced command line based metronome for JACK'
+arch=(x86_64 aarch64)
+url='http://das.nasophon.de/klick/'
```

Comment:
Should we put the URL of the fork here?

Reply:
I'm not sure. The documentation is still on the original homepage. OTOH, the github page has a link to that. Can you decide? :)